spec/formatting.md

```diff
@@ -146,6 +153,8 @@ and different implementations MAY choose to perform different levels of resoluti
 >   getValue(): unknown
 >   resolvedOptions(): { [key: string]: MessageValue }
 >   selectKeys(keys: string[]): string[]
+>   directionality() : Dir // where Dir is an enum type, one of « `'LTR'`, `'RTL'`, `'unknown'` »
```

Comment:
We don't need to define a new type in order to communicate the intent here; this is TypeScript after all.
```suggestion
>   directionality(): 'LTR' | 'RTL' | 'unknown'
```